apps/files_sharing/src/components/SharingEntryLink.vue

```diff
@@ -424,6 +424,11 @@ export default {
 					})
 				}
 				if (this.share.label && this.share.label.trim() !== '') {
+					if (this.isEmailShareType) {
+						return t('files_sharing', 'Mail share ({label})', {
```

Comment:
```suggestion
						return t('files_sharing', '{label} (mail share)', {
```

Does that make sense? Because I set a label and I would like to have the highlight on my label and not on "Mail share"

Reply:
Without label:

![Bildschirmfoto 2021-03-18 um 13 49 23](https://user-images.githubusercontent.com/245432/111628695-c9a70880-87f0-11eb-9c22-ac09692b57cb.png)

With label "My label":

![Bildschirmfoto 2021-03-18 um 13 49 32](https://user-images.githubusercontent.com/245432/111628724-d3c90700-87f0-11eb-8195-1936a165711a.png)

Reply:
Proposed change:

![Bildschirmfoto 2021-03-18 um 13 50 42](https://user-images.githubusercontent.com/245432/111628860-f6f3b680-87f0-11eb-94a4-4e1f969aaadc.png)

Reply:
I used the same syntax as the link share. Maybe let's fix it in followup then if Jan wants it :)